page.go

```diff
@@ -217,6 +217,43 @@ type packageData struct {
 	GitTreeName    string
 }
 
+type cacheEntry struct {
+	t  time.Time
```

Comment:
I'd be inclined to call this timestamp or ts.

Reply:
I've gone with Timestamp here.